indico/modules/events/registration/templates/emails/base_registration_details.html

```diff
@@ -36,21 +49,73 @@ <h2>{{ section.title }}</h2>
 {% macro render_regdata(regdata) %}
     <dd style="width: 70%; float: left; margin: 0; min-height: 20px; white-space: pre-wrap; word-break: break-all; padding-left: 20px; box-sizing: border-box;">
         {%- set friendly_data = regdata.friendly_data -%}
-        {%- if regdata.field_data.field.input_type == 'accommodation' and friendly_data -%}
-            {% if friendly_data['is_no_accommodation'] %}
-                {{- friendly_data.choice -}}
-            {% else %}
-                {{- friendly_data.choice }} ({{ friendly_data.arrival_date | format_date(locale='en_GB') }} - {{ friendly_data.departure_date | format_date(locale='en_GB') -}})
-            {% endif %}
-        {%- elif regdata.field_data.field.input_type == 'multi_choice' -%}
-            {{- friendly_data | join(', ') }}
-        {%- else -%}
-            {{- friendly_data -}}
-        {%- endif -%}
+        {%- set type = regdata.field_data.field.input_type -%}
+        <span>
+            {{- render_friendly_data(friendly_data, type) -}}
+        </span>
         {%- if regdata.price -%}
-            <span style="float: right; font-style: italic; font-size: 0.9em;">
-                {{- regdata.render_price() -}}
+            {{- render_price(regdata) -}}
+        {%- endif -%}
+    </dd>
+{% endmacro %}
+
+
+{% macro render_modified_regdata(regdata, diff) %}
+    <dd style="width: 70%; float: left; margin: 0; min-height: 20px; white-space: pre-wrap; word-break: break-all; padding-left: 20px; box-sizing: border-box;">
+        {%- set type = regdata.field_data.field.input_type -%}
+        {%- set old = diff.old -%}
+        {%- set new = diff.new -%}
+        {%- set old_friendly_data = render_friendly_data(old.friendly_data, type) -%}
+        {%- set new_friendly_data = render_friendly_data(new.friendly_data, type) -%}
+        <span>{#- -#}
+            {%- if old_friendly_data -%}
+                <s style="margin-right: 5px;">{{ old_friendly_data }}</s>{#- -#}
+            {%- endif -%}
+            {%- if new_friendly_data -%}
+                <span style="background-color: yellow;">{{ new_friendly_data }}</span>{#- -#}
+            {%- endif -%}
+        </span>
+        {%- if old.price != new.price -%}
+            <span style="float: right; font-style: italic; font-size: 0.9em;">{#- -#}
+                <s style="margin-right: 5px;">{{- regdata._render_price(old.price) -}}</s>{#- -#}
+                <span style="background-color: yellow;">{{- regdata._render_price(new.price) -}}</span>{#- -#}
```

Comment:
We use it without a space almost everywhere else as well

```suggestion
            <span style="float: right; font-style: italic; font-size: 0.9em;">{#--#}
                <s style="margin-right: 5px;">{{- regdata._render_price(old.price) -}}</s>{#--#}
                <span style="background-color: yellow;">{{- regdata._render_price(new.price) -}}</span>{#--#}
```